src/drivers/boards/px4fmu-v3/board_config.h

```diff
@@ -0,0 +1 @@
+../px4fmu-v2/board_config.h
```

Comment:
Why can't you use that path here just as an include? I don't see the need for a symbolic link here.